### ISSUE TITLE
Is this repo still active?

### ISSUE THREAD

**Issue by moelleer** [10up/elasticpress-autosuggest#4]
Hi!

I'm going to build an autocompletion plugin for Elasticpress and found this plugin, but it doesn't seem to work for me. Is this plugin still being maintained or should I look for another alternative?

Thanks!
Johan

**Comment by AaronHolbrook** [comment]
Hi @moelleer - feel free to submit a PR if you find something that needs fixing. Last I checked this plugin worked well.

Can you describe how you tried to use it and how it failed?

**Comment by moelleer** [comment]
I just added the plugin, activated it, and added the elasticpress-autosuggest class to the search input. Is there anything else I should do to make it work?

**Comment by ShariqKhan2012** [comment]
@moelleer I just updated something on https://github.com/10up/ElasticPress-Autosuggest/issues/3

Might be useful for you.

@AaronHolbrook I could not find any method anywhere in code to pick up the EP_HOST from wp-config.php, though there seems to be a line which could be relevant here:

`var $dataESHost = $epInput.data( 'es-host' );`

This seems to suggest one also needs to add **es-host-data** attribute to the search input.
Not sure if I am doing it correct. Can you please comment?

Btw, thank you for the awesome plugin that is ElasticPress :+1:

**Comment by mikedamoiseau** [comment]
@moelleer Yes, you also have to add this html markup near your search input field:

```html
<div class="ep-autosuggest"><ul class="autosuggest-list"></ul></div>
```

For more details, you can have a look at the 10up#3
